app/actions/UploadInventory.js

```diff
@@ -239,6 +238,9 @@ const getBodyData = (inventory) => {
     if (inventory.tagId) {
       bodyData.tag = inventory.tagId;
     }
+    if (inventory.projectId) {
+      bodyData.plantProject = inventory.projectId;
```

Comment:
According to the documentation at https://plant-for-the-planet.stoplight.io/docs/treecounter-platform/reference/TreeMapper.yaml/paths/~1plantLocations/post the parameter should be named `project`.

Reply:
Returns 400 error, if I use `project` as property instead of `plantProject`
`"errors": [
            "This form should not contain extra fields."
        ],`

If I use `plantProject` with different id not registered it returns this error. So the api looks working but yes it does not return any project in response.
`"plantProject": {
                "errors": [
                    "The provided plant project does not belong to the authenticated user."
                ]
            },`

Reply:
@jmiridis can you please help with the API?